build.gradle

```diff
@@ -35,6 +35,3 @@ allprojects {
     }
 }
 
-task clean(type: Delete) {
-    delete rootProject.buildDir
```

Comment:
Why?

Reply:
It's needed. Or else it throws duplicate task error

Reply:
I was getting this error "Cannot add task 'clean' as a task with that name already exists." after changing the version of spotless.